postgresql/benchmark.sh

```diff
@@ -1,35 +1,51 @@
 #!/bin/bash
 
+set -eux
+
+PGVERSION=17
+
 sudo apt-get update
-sudo apt-get install -y postgresql-common
-sudo apt-get install -y postgresql-14
+sudo apt install -y postgresql-common
```

Comment:
Perhaps we should un-install existing Postgres first?

Reply:
I don't think this necessary, you can install multiple PG versions at the same time. `postgresql-common` is primarily needed to get this install script.